go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/OctopusDeploy/terraform-provider-octopusdeploy
 go 1.21
 
 require (
-	github.com/OctopusDeploy/go-octopusdeploy/v2 v2.60.0
+	github.com/OctopusDeploy/go-octopusdeploy/v2 v2.62.3-0.20241202024636-075a4f06c227
```

Comment:
Use https://github.com/OctopusDeploy/go-octopusdeploy/pull/288 for now.

Will update with the correct version once go client is merged after octopus server changes.